Update Cert Install Steps for latest iOS (v15.3.1) (#3737)
cloudflare/cloudflare-docs: content/cloudflare-one/connections/connect-devices/warp/install-cloudflare-cert.md

> The device will show a message: _This website is trying to open Settings to how you a configuration profile. Do you want to allow this?_

Reviewer: ```suggestion
The device will show a message: _This website is trying to download a configuration profile. Do you want to allow this?_
```

![iOS download](/cloudflare-one/static/documentation/connections/ios_cert_download.PNG)

2.  Tap **Allow**.

> 3.  Navigate to **Settings** > **General** > **VPN & Device Management** and the new downloaded profile should be listed.

Reviewer: ```suggestion
3.  Navigate to **Settings** > **General** > **VPN & Device Management** and find the **Cloudflare for Teams ECC Certificate Authority** profile.
```

{{<Aside type="note">}}

> Alternatively you can navigate to **Settings** and a new **Profile Downloaded** section should be shown directly beneath your iCloud user account info.

Reviewer: ```suggestion
Alternatively you can navigate to **Settings**, where a new **Profile Downloaded** section will appear directly beneath your iCloud user account info.
```
